scripts/2500/stage/demux-2500.bash

```diff
@@ -0,0 +1,141 @@
+date > ${IN_DIR}/demuxstarted.txt
+cp ${IN_DIR}/SampleSheet.csv ${IN_DIR}/Data/Intensities/BaseCalls/SampleSheet.csv
+
+# here we go!
+log "Setup correct, starts demuxing . . ."
+
+echo "get basemask ${IN_DIR}"
+BASEMASK=$(demux basemask create --application wes --lane 1 ${IN_DIR})
+UNALIGNED_DIR=Unaligned-${BASEMASK//,}
+
+# DEMUX !
+JOB_TITLE=Demux_${RUN}
+log "sbatch --wait -A ${SLURM_ACCOUNT} -J ${JOB_TITLE} -o ${PROJECTLOG} ${SCRIPT_DIR}/demux-2500.sh ${IN_DIR} ${OUT_DIR}/${RUN} ${BASEMASK} ${UNALIGNED_DIR}"
+RES=$(sbatch --wait -A ${SLURM_ACCOUNT} -J ${JOB_TITLE} -o ${PROJECTLOG} ${SCRIPT_DIR}/demux-2500.sh ${IN_DIR} ${OUT_DIR}/${RUN} ${BASEMASK} ${UNALIGNED_DIR})
+
+log "bcl2fastq finished!"
+
+# Add samplesheet to unaligned folder
+cp ${IN_DIR}/SampleSheet.csv ${OUT_DIR}/${RUN}/${UNALIGNED_DIR}/
+
+# Restructure the output dir!
+FC=${RUN##*_}
+FC=${FC:1}
+shopt -s nullglob
+for PROJECT_DIR in ${OUT_DIR}/${RUN}/${UNALIGNED_DIR}/*; do
+    if [[ ! -d ${PROJECT_DIR} ]]; then continue; fi
+
+    PROJECT=$(basename ${PROJECT_DIR})
+    if [[ ${PROJECT} == 'Stats' ]]; then continue; fi
+    if [[ ${PROJECT} == 'Reports' ]]; then continue; fi
+    if [[ ${PROJECT} =~ Project_* ]]; then continue; fi
+    if [[ ${PROJECT} == 'indexcheck' ]]; then
+        mv ${PROJECT_DIR} ${OUT_DIR}/${RUN}/${UNALIGNED_DIR}/Project_${PROJECT}
+        continue
+    fi
+
+    for SAMPLE_DIR in ${PROJECT_DIR}/*; do
+        for FASTQ_FILE in ${SAMPLE_DIR}/*; do
+            FASTQ=$(basename ${FASTQ_FILE})
+            SAMPLE=$(basename ${SAMPLE_DIR})
+            LANE_READ_PART=${FASTQ#*_*_}
+            INDEX="NNNNNNNN-NNNNNNNN"
+            log "mv ${FASTQ_FILE} ${SAMPLE_DIR}/${SAMPLE}_${INDEX}_${LANE_READ_PART}"
+            mv ${FASTQ_FILE} ${SAMPLE_DIR}/${SAMPLE}_${INDEX}_${LANE_READ_PART}
+        done
+
+        SAMPLE=$(basename ${SAMPLE_DIR})
+        log "mv ${SAMPLE_DIR} ${PROJECT_DIR}/Sample_${SAMPLE}"
+        mv ${SAMPLE_DIR} ${PROJECT_DIR}/Sample_${SAMPLE}
+    done
+
+    log "mv ${PROJECT_DIR} ${OUT_DIR}/${RUN}/${UNALIGNED_DIR}/Project_${PROJECT}"
+    mv ${PROJECT_DIR} ${OUT_DIR}/${RUN}/${UNALIGNED_DIR}/Project_${PROJECT}
+done
+
+# Add stats
+log "cgstats add --machine 2500 --unaligned ${UNALIGNED_DIR} ${OUT_DIR}/${RUN}/"
+cgstats add --machine 2500 --unaligned ${UNALIGNED_DIR} ${OUT_DIR}/${RUN}/ &>> ${PROJECTLOG}
+
+# create stats files
+FC=$(echo ${RUN} | awk 'BEGIN {FS="/"} {split($(NF),arr,"_");print substr(arr[4],2,length(arr[4]))}')
+PROJs=$(ls ${OUT_DIR}/${RUN}/${UNALIGNED_DIR}/ | grep Proj)
+for PROJ in ${PROJs[@]}; do
+    prj=$(echo ${PROJ} | sed 's/Project_//')
+    log "cgstats select --project ${prj} ${FC} &> ${OUT_DIR}/${RUN}/stats-${prj}-${FC}.txt"
+    cgstats select --project ${prj} ${FC} &> ${OUT_DIR}/${RUN}/stats-${prj}-${FC}.txt
+done
+
+log "rm -f ${OUT_DIR}/${RUN}/copycomplete.txt"
+rm -f ${OUT_DIR}/${RUN}/copycomplete.txt
+
+log "date > ${OUT_DIR}/${RUN}/demuxcomplete.txt"
+date > ${OUT_DIR}/${RUN}/demuxcomplete.txt
```

Comment:
add new line

Reply:
To the end of the file*

Reply:
fixed in https://github.com/Clinical-Genomics/demultiplexing/pull/144/commits/73cb21e670113ab1eedd0fcbe2100b902f8ca256